src/types/markers.js

```diff
@@ -268,10 +268,22 @@ export type InvalidationPayload = {
 export type NetworkPayload = {
   type: 'Network',
   URI?: string,
+  RedirectURI?: string,
   id: number,
+  pri: number,
+  count?: number,
```

Comment:
I made a comment here that got hidden by Github because you change this line for another reason. (this is something quite painful with github).

I'll add it back:

> can you please add some comments about what these 2 properties mean?

> Also, are they always present in type == 'Network' markers, including in some previous builds of Firefox ? If not, we need to specify them with a question mark. If there were always present since the first version of the Network markers, then it's fine like this.

I see you added the question mark for `count` but not for `pri`, is that because `pri` has always been present ?

Reply:
Yes, I believe it has always been present (and should be, as priority can change after Start, so we include it in any later markers for the load - redirect or stop)